src/default_theme/index.js

```diff
@@ -9,6 +9,15 @@ var fs = require('fs'),
   LinkerStack = require('../').util.LinkerStack,
   hljs = require('highlight.js');
 
+function is_function(section) {
```

Comment:
Mind defining this as `isFunction`? camelCase is the default naming convention in documentation.js